code/datums/station_traits/neutral_traits.dm

```diff
@@ -0,0 +1,87 @@
+/datum/station_trait/bananium_shipment
+	name = "Bananium Shipment"
+	trait_type = STATION_TRAIT_NEUTRAL
+	weight = 5
+	report_message = "Rumors has it that the clown planet has been sending support packages to clowns in this system"
+	trait_to_give = STATION_TRAIT_BANANIUM_SHIPMENTS
+
+/datum/station_trait/bananium_shipment
+	name = "Tranquilite Shipment"
+	trait_type = STATION_TRAIT_NEUTRAL
+	weight = 5
+	report_message = "Rumors has it that the mime federation has been sending support packages to mimes in this system"
+	trait_to_give = STATION_TRAIT_TRANQUILITE_SHIPMENTS
+
+/datum/station_trait/unique_ai
+	name = "Unique AI"
+	trait_type = STATION_TRAIT_NEUTRAL
+	weight = 5
+	show_in_report = TRUE
+	report_message = "For experimental purposes, this station AI might show divergence from default lawset. Do not meddle with this experiment, we've removed \
+		access to your set of alternative upload modules because we know you're already thinking about meddling with this experiment."
+	trait_to_give = STATION_TRAIT_UNIQUE_AI
+	blacklist = list(/datum/station_trait/random_event_weight_modifier/ion_storms)
+
+/datum/station_trait/unique_ai/on_round_start()
+	. = ..()
+	for(var/mob/living/silicon/ai/ai as anything in GLOB.ai_list)
```

Comment:
Does this actually remove the upload modules as stated in the description?

Reply:
Yep

Reply:
Doesn't remove the smartfridges however, fixed